azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/VirtualMachineImpl.java

```diff
@@ -1634,6 +1758,52 @@ public StorageAccount call(StorageAccount storageAccount) {
         return Observable.just(null);
     }
 
+    private Observable<StorageAccount> handleBootDiagnosticsStorageSettings(StorageAccount diskStorageAccount) {
+        final Observable<StorageAccount> diskStgObservable = Observable.just(diskStorageAccount);
+        if (this.inner().diagnosticsProfile() == null
```

Comment:
since these are a bunch of ifs that are alternatives to each other, it'd be more readable I think if these were explicitly `else if` not just `if`.